dotCMS/src/main/java/com/dotcms/contenttype/business/ContentTypeAPI.java

```diff
@@ -254,5 +255,5 @@
       throws DotDataException;
 
 
-
+  Map<String, Long> getEntriesByContentTypes() throws DotDataException;
```

Comment:
We are missing javadoc here explaining what the getEntriesByContentTypes method does and what is returning